api-reference/beta/api/alert_update.md

```diff
@@ -0,0 +1,162 @@
+# Update alert
+
+Update an editable **alert** property within any integrated solution to keep alert status and assignments in sync across solutions. This method updates any solution that has a record of the referenced alert ID. 
+
+>**Note:** You must include the alert ID as a parameter with this method.
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |   SecurityEvents.ReadWrite.All  |
+|Delegated (personal Microsoft account) |  Not supported  |
+|Application | SecurityEvents.ReadWrite.All |
+
+## HTTP request
+
+This method must include the alert ID as a parameter.

```

Comment:
We state this in the overview; do we need to repeat here? It should be clear from the syntax shown.